crates/starknet/src/starknet/add_deploy_account_transaction.rs

```diff
@@ -121,7 +125,8 @@ mod tests {
         let txn = starknet.transactions.get_by_hash_mut(&txn_hash).unwrap();
 
         assert_eq!(txn.finality_status, None);
-        assert!(txn.execution_result.revert_reason().unwrap().contains("Fee transfer failure"));
+        println!("{}", txn.execution_result.revert_reason().unwrap());
```

Comment:
do we need to print this?

Reply:
https://github.com/0xSpaceShard/starknet-devnet-rs/commit/8fbec9cd3cbfa72e68c968b3e774e8a93d363292